data/neo4j/neo4j_database.py

```diff
@@ -0,0 +1,60 @@
+from neo4j import GraphDatabase
+from typing import Optional
+
+query_periodic_commit = "USING PERIODIC COMMIT 10000 "
+
+
+class Database:
+    def __init__(self, uri: str, user: str, password: str):
+        self.driver = GraphDatabase.driver(uri, auth=(user, password))
+
+    def close(self):
+        self.driver.close()
+
+    """ GET """
+
+    @staticmethod
+    def get_node(node_id: int, node_type: str, no_return: bool = False) -> str:
```

Comment:
should this say something like `query_node`? as it doesn't get you the node, it only constructs the query

Reply:
Makes sense, will change it.